### PR TITLE
ES6 Porting : 'Keyed Fragments' (10.6 Create Fragments)

### PR DESCRIPTION
**Pull request by ankitml**
Reference Issue : https://github.com/facebook/react/issues/3710 
*Before* submitting a pull request, please make sure the following is done...

1. Fork the repo and create your branch from `master`.
2. If you've added code that should be tested, add tests!
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes (`grunt test`).
5. Make sure your code lints (`grunt lint`) - we've done our best to make sure these rules match our internal linting guidelines.
6. If you haven't already, complete the [CLA](https://code.facebook.com/cla).

### CODE REVIEW

File: docs/docs/10.6-create-fragment.md [hunk 1]
```diff
@@ -11,24 +11,21 @@ In most cases, you can use the `key` prop to specify keys on the elements you're
...
-  },
-  render: function() {
-    var children;
-    if (this.props.swapped) {
-      children = [this.props.rightChildren, this.props.leftChildren];
-    } else {
-      children = [this.props.leftChildren, this.props.rightChildren];
-    }
-    return <div>{children}</div>;
+const Swapper  = ({props}) {
```
Comment (roganov) [inline docs/docs/10.6-create-fragment.md]: 1) should be `(props)`
2) forgot arrow

File: docs/docs/10.6-create-fragment.md [hunk 2]
```diff
@@ -11,24 +11,21 @@ In most cases, you can use the `key` prop to specify keys on the elements you're
...
+  const children;
```
Comment (roganov) [inline docs/docs/10.6-create-fragment.md]: `const`s must be initialized

File: docs/docs/10.6-create-fragment.md (hunk 1 above)
Comment (jimfb) [inline docs/docs/10.6-create-fragment.md]: @roganov is correct, but let's just use an es5 function for this component.  It is more familiar to newbies coming to React.

**Comment by gaearon** [comment]
Thank you so much!

I rebased and merged this as a6445a18f98c9b5c453acee7fb5783b35603598e, keeping the commit credited to you.

Cheers!

**Pull request closed without merging**